recipes-support/landing-zone/landing-zone_0.3.0.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "Landing Zone - DRTM Tool"
+HOMEPAGE = "https://github.com/TrenchBoot/landing-zone"
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=4641e94ec96f98fabc56ff9cc48be14b"
+
+SRC_URI = "git://github.com/TrenchBoot/landing-zone.git;protocol=https;branch=master"
+SRCREV = "5073f638cb34ab3b3b96a6f0e67a298e043b1ecd"
+
+RDEPENDS_${PN} = "bash"
+
+S = "${WORKDIR}/git"
+
+EXTRA_OEMAKE += "CC=gcc"
```

Comment:
@Asiderr We should not use the native gcc. We should use the CC from Yocto.

Reply:
@macpijan Compilation with CC from the Yocto fails:
```
| /work/build/tmp/work/dbft3b-pce-linux/landing-zone/0.3.0-r0/git/sha1sum.c:147: undefined reference to `__stack_chk_fail'
| /work/build/tmp/work/dbft3b-pce-linux/landing-zone/0.3.0-r0/recipe-sysroot-native/usr/bin/x86_64-pce-linux/../../libexec/x86_64-pce-linux/gcc/x86_64-pce-linux/9.2.0/ld: sha1sum.o: in function `sha1sum':
| /work/build/tmp/work/dbft3b-pce-linux/landing-zone/0.3.0-r0/git/sha1sum.c:239: undefined reference to `__stack_chk_fail'
| /work/build/tmp/work/dbft3b-pce-linux/landing-zone/0.3.0-r0/recipe-sysroot-native/usr/bin/x86_64-pce-linux/../../libexec/x86_64-pce-linux/gcc/x86_64-pce-linux/9.2.0/ld: main.o: in function `lz_main':
| /work/build/tmp/work/dbft3b-pce-linux/landing-zone/0.3.0-r0/git/main.c:247: undefined reference to `__stack_chk_fail'
```
That hack was provided by @miczyg1 .

Reply:
That's not a hack. The environment I was building the LZ was lacking CC environment variable required by Makefile. As @macpijan stated, native Yocto CC should be used. If CC is already set, the whole EXTRA_OEMAKE is obsolete

Reply:
@macpijan Solved.